content/05_labs/02_org_setup/_index.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: "AWS Organization"
+title: "Organizing accounts"
```

Comment:
Don't write headlines with present participle! (I just made up that rule)

Reply:
'Present' is hands-down the best participle but i have changed it to "Account Organization" for now.